server/athenian/api/internal/features/entries.py

```diff
@@ -1690,7 +1688,6 @@ async def _calc_pull_request_facts_github(
         else:
             miner, unreleased_facts, matched_bys, unreleased_prs_event = await tasks[0]
         new_deps = miner.dfs.deployments.copy()
-        new_jira = miner.dfs.jiras.index.copy()
```

Comment:
This df can mutate asynchronously. It is super important to make a copy here and use it later. I remember spending several hours on debugging the problem last year.